pkg/crc/machine/machine.go

```diff
@@ -726,39 +745,35 @@ func GetProxyConfig(machineName string) (*network.ProxyConfig, error) {
 
 // Return console URL if the VM is present.
 func GetConsoleURL(consoleConfig ConsoleConfig) (ConsoleResult, error) {
-	result := &ConsoleResult{Success: true}
 	// Here we are only checking if the VM exist and not the status of the VM.
 	// We might need to improve and use crc status logic, only
 	// return if the Openshift is running as part of status.
 	libMachineAPIClient := libmachine.NewClient(constants.MachineBaseDir, constants.MachineCertsDir)
 	host, err := libMachineAPIClient.Load(consoleConfig.Name)
 
 	if err != nil {
-		result.Success = false
-		result.Error = err.Error()
-		return *result, errors.New(err.Error())
+		return ConsoleResult{Error: err.Error()}, errors.New(err.Error())
```

Comment:
Same comment here. This change looks more 'go-ish' to me, but I don't think it's required in this already huge change (because of the vendoring), and it's also inconsistent with the way all other methods return their result.